service/src/main/java/tds/config/exceptions/NotFoundException.java

```diff
@@ -0,0 +1,10 @@
+package tds.config.exceptions;
+
+/**
+ * Created by jjohnson on 9/1/16.
+ */
+public class NotFoundException extends Exception {
```

Comment:
I prefer us to use RuntimeExceptions instead of checked Exceptions unless there is a specific reason we want to use checked Exceptions.  The reason is I don’t want to force upon a caller how to handle a failure. What I instead prefer is Spring’s take.  They throw runtime but document everything that can be thrown and the reason why it is thrown.  It is then up to the caller to determine if they want to catch and handle.  For example, jdbcTemplate.queryForObject.  DataAccessException is runtime. http://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/jdbc/core/namedparam/NamedParameterJdbcTemplate.html#queryForObject-java.lang.String-java.util.Map-java.lang.Class-